### PR TITLE
Progressive back-off.

### PR DESCRIPTION
**Pull request by bradserbuddy**

### CODE REVIEW

**Comment by bradserbuddy** [comment]
Ping!

File: Samples/BuddyStarter/BuddyStarter/BPStarterAppDelegate.m [hunk 1]
```diff
@@ -30,6 +30,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     // This initialization does NOT cause a network call, but will be processed
     // upon the first call to a Buddy API.
 
+    #pragma message ("Go to http://buddyplatform.com to get an app ID and app key.")
```
Comment (cacsar) [inline Samples/BuddyStarter/BuddyStarter/BPStarterAppDelegate.m]: So this is to give a compiler message if the developer doesn't do things?

File: Src/Lib/BuddySDK/BPCrashManager.m [hunk 2]
```diff
@@ -339,7 +339,7 @@ - (void)sendCrashReports:(PLCrashReport *)crashReport onSuccess:(void (^)())onSu
             }
         }
     } else {
-        // Crap. There was something lower level, like overreleasing an object. Try to build the crash info from the thread list.
```
Comment (cacsar) [inline Src/Lib/BuddySDK/BPCrashManager.m]: Good removal.

**Comment by cacsar** [comment]
Looks reasonable enough, but then I'm not super familiar with our SDK code and it's been 6? years since I last did anything on iOS.

File: Samples/BuddyStarter/BuddyStarter/BPStarterAppDelegate.m (hunk 1 above)
Comment (bradserbuddy) [inline Samples/BuddyStarter/BuddyStarter/BPStarterAppDelegate.m]: Yup.

**Pull request merged**